### ISSUE TITLE
Result.php code, as written, requires PHP 5.4

### ISSUE THREAD

**Issue by micmath** [fortrabbit/datafilter#2]
I get

```
Parse error: syntax error, unexpected '[' in php/vendor/datafilter/datafilter/src/DataFilter/Result.php on line 437
```

which has this code:

``` php
$this->validAttribs[$attribName] = ['value' => $default, 'attrib' => &$attrib];
```

I'm using PHP 5.3.15 and [your project's github homepage](https://github.com/fortrabbit/datafilter/) says "PHP 5.3 is required" strongly implying that it does not require 5.4. But according to [the PHP.net docs](http://php.net/manual/en/language.types.array.php) "As of PHP 5.4 you can also use the short array syntax, which replaces array() with []."

Something needs to be clarified here.

**Comment by ukautz** [comment]
Yes, you are right, the tests are still 5.4. Will downgrade them soon to be compatible with 5.3.
